lms/services/lti_h.py

```diff
@@ -22,41 +23,84 @@ def __init__(self, _context, request):
         self._lti_user = request.lti_user
         self._h_user = request.lti_user.h_user
 
+        self._authority = request.registry.settings["h_authority"]
```

Comment:
It's a shame that the authority setting is needed in both `LTIHService` and `HAPI`